src/CSExitPenalties.sol

```diff
@@ -0,0 +1,206 @@
+// SPDX-FileCopyrightText: 2025 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.24;
+
+import { SafeCast } from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+import { Math } from "@openzeppelin/contracts/utils/math/Math.sol";
+
+import { ICSAccounting } from "./interfaces/ICSAccounting.sol";
+import { ICSExitPenalties, MarkedUint248 } from "./interfaces/ICSExitPenalties.sol";
+import { ExitPenaltyInfo } from "./interfaces/ICSExitPenalties.sol";
+import { ICSModule } from "./interfaces/ICSModule.sol";
+import { ICSParametersRegistry } from "./interfaces/ICSParametersRegistry.sol";
+import { Initializable } from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+contract CSExitPenalties is ICSExitPenalties, Initializable {
+    using SafeCast for uint256;
+
+    uint256 public constant VOLUNTARY_EXIT_TYPE_ID = 0;
```

Comment:
Can you fetch this value from the other contract?

Reply:
i'd expect to prefetch this value from some core contract, it's not there yet